script/import_scripts/phpbb3/support/text_processor.rb

```diff
@@ -197,5 +197,11 @@ def fix_markdown(text)
       text.gsub!(/^!\[[^\]]*\]\([^\]]*\)$/i) { |img| "\n#{img.strip}\n" } # space out images single on line
       text
     end
+
+    def process_videos(text)
+      # [YOUTUBE]<id>[/YOUTUBE]
+      text.gsub(/\[youtube\](.+?)\[\/youtube\]/i) { "https://www.youtube.com/watch?v=#{$1}" }
```

Comment:
Shouldn't we ensure this is on a separate line like the method above does for other tags? Otherwise the video won't be oneboxed.

Reply:
yes, I'll add the break of lines, thanks!